tests/functional/search-auth.js

```diff
@@ -198,6 +198,34 @@ registerSuite("Search (auth)", {
         .end();
     },
 
+    "Label have correct URL after search without results"() {
+      // load up a garbage search, so we can more easily detect when
+      // the search values we want are loaded.
+      return FunctionalHelpers.openPage(
+        this,
+        url(
+          "/issues"
+        ),
+        ".grid"
+      )
+        .findByCssSelector("#js-SearchForm-input")
+        .click()
+        .type("noExpectedResult123")
+        .submit()
```

Comment:
I think we can get rid of this `submit()` too, since you're submitting the form just after.